dist/media/package.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
Hmm, just tested it, and it looks like it is working, but I'm not 100% sure why... ;)